src/NHSD.BuyingCatalogue.Documents.API/Controllers/SolutionsController.cs

```diff
@@ -11,12 +12,16 @@ namespace NHSD.BuyingCatalogue.Documents.API.Controllers
     [AllowAnonymous]
     public class SolutionsController : ControllerBase
     {
+        private readonly IStorage _storage;
+
+        public SolutionsController(IStorage storage)
+            => _storage = storage;
+
         [HttpGet]
         [Route("{id}/documents/")]
-        [ProducesResponseType(typeof(IEnumerable<string>), (int)HttpStatusCode.OK)]
-        public ActionResult GetFileNames(string id)
-        {
-            return Ok(new List<string>{"roadmap.pdf"});
-        }
+        [ProducesResponseType(typeof(IAsyncEnumerable<string>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.InternalServerError)]
+        public ActionResult GetFileNames(string id) 
```

Comment:
I was expecting a view model here. I assume as the requirements grow we will introduce this?